lineapy/db/db.py

```diff
@@ -470,6 +470,24 @@ def node_value_in_db(
             .exists()
         ).scalar()
 
+    def number_of_artifacts_per_node(
```

Comment:
Do we know whether a new pickle gets created when the .save is called on the same node multiple times?